general/README.md

```diff
@@ -208,15 +209,102 @@ Avoid variable indirection when possible. Reassigning variables when no transfor
 ```js
 // good
 _handleEvent = function(e) {
-    _doSomethingWithEvent(e.target.checked);   
+    _doSomethingWithEvent(e.target.checked);
 }
 
 // bad
 _handleEvent = function(e) {
     var checked = e.target.checked;
 
-    _doSomethingWithEvent(checked);   
+    _doSomethingWithEvent(checked);
 }
 ```
 
 **[⬆ back to top](#table-of-contents)**
+
+## Naming Conventions
+> There are only two hard things in Computer Science: cache invalidation and naming things. *Phil Karlton*
+
+### Variables: Classes or Newable Functions (meant to be factories).
+It should be singular and begin with a capital letter.
+
+```js
+var ticket = new Ticket(id: 555);
```

Comment:
you should have bad examples as well. Maybe one with the class name not be capitalized and another that's snake_case?

Also, just to prevent confusion, name the variable to `myTicket`?